### PR TITLE
Merging all the work from Secondary to master branch for code review evaluation

### PR DESCRIPTION
**Pull request by memelopez**
we started the project yesterday because of a confusion of ours, it still has to improve but we are submitting it for the sof deadline. 👍🏼

### CODE REVIEW

**Comment by memelopez** [comment]
Our page looks crisp, almost perfect. We have no linter errors and are ready for another code review.

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```
Comment (krys2fa) [inline README.md]: - [x] [REQUIRED] Please provide the steps to take in order for the project to be setup. If no steps are required, you should remove these headings.

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Nicely done @memelopez  and @Fondem-Jr  👏 
Your project displays good use of semantic HTML as well as CSS classes. 💪 
You have a professional-looking README file as well. 👍 

Just a few more adjustments to complete the project. They can be seen below as well as in the comments:
### Status: Changes Required ♻️ 

**NB:** Review was done at a screen size of **1440px**. Kindly state your preferred screen size if your clone displays better on your machine.

- [x] [REQUIRED] Elements should be aligned with a combination of all three techniques ([float](
https://css-tricks.com/all-about-floats/), [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [grid](https://css-tricks.com/snippets/css/complete-guide-grid/)). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_pathwright.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/)) and ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_odin.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements#introduction)). You need to use float as well.

- [x] [OPTIONAL] It should have a fixed footer (i.e it should stick to the bottom part of the screen) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))

Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Comment by memelopez** [comment]
please review at a screen size of 1700px
The page uses:
- floats in the top right left plus icon in the 'newsletter' section
- position fixed on navbar and footer
- grid on the articles of the 'newsletter' and in the 'theory of inflation' section
- flexbox all throughout the layout of the page
- position absolute, relative different sections

File: css/style-storybody.css [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+  font-weight: 500;
+  border-bottom: none;
+  padding-bottom: 0;
+  font-family: georgia, "times new roman", Times, serif;
+  font-size: 14px;
+  line-height: 17px;
+  color: rgb(51, 51, 51);
+}
+
+/* #foraside {
```
Comment (umairarshadbutt) [inline css/style-storybody.css]: Remove commented CSS

File: css/style-storybody.css [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+  display: flex;
+  flex-direction: row;
+} */
+
+.storycompanion {
+  /* width: 1200px; */
```
Comment (umairarshadbutt) [inline css/style-storybody.css]: Remove commented CSS

File: css/styles-bottomNav.css [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+  color: #666;
+  font-family: helvetica, arial, sans-serif;
+  padding: 10px 0;
+  text-decoration: none;
+  white-space: nowrap;
+  font-size: 11px;
+}
+
+.secondHRZTLnav {
+  /* display: inline-block; */
```
Comment (umairarshadbutt) [inline css/styles-bottomNav.css]: Remove commented CSS

File: css/styles-lowerFooter.css [hunk 5]
```diff
@@ -0,0 +1,48 @@
+/* Styles for the lowerEST footer */
+
+.main-footer {
+  /* transform: translateY(-56px); */
```
Comment (umairarshadbutt) [inline css/styles-lowerFooter.css]: Remove commented CSS

File: css/styles-mainHeader.css [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+  line-height: 11px;
+  font-weight: 700;
+  letter-spacing: 0.05em;
+  text-transform: uppercase;
+  padding: 11px 12px 8px;
+  margin-right: 7px;
+}
+
+.logIn {
+  /* margin-right: 7px; */
```
Comment (umairarshadbutt) [inline css/styles-mainHeader.css]: Remove commented CSS

File: index.html [hunk 7]
```diff
@@ -0,0 +1,1224 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NYT</title>
+    <link rel="stylesheet" href="css/style-storybody.css">
+    <link rel="stylesheet" href="css/styles-mainHeader.css">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="css/styles-headerArticle.css">
+    <link rel="stylesheet" href="css/styles-lowerFooter.css">
+    <link rel="stylesheet" href="css/styles-bottomNav.css">
+    <link rel="stylesheet" href="css/styles-recirc.css">
```
Comment (umairarshadbutt) [inline index.html]: `href="css/styles-recirc.css"`  is not a correct path please use like `href="./css/styles-recirc.css"` (**update all links**)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @memelopez 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Remove commented CSS from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [x] MIT License giving 404 error. Please fix it.
- [x] Update css path. see inline comments
- [x] Please implement the previous TSE's suggestions about the README file
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 

Great job @team, lovely clone I liked it.

now you can merge your project is approved :checkered_flag: 

## Happy Coding :tophat:

**Pull request merged**